web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -201,3 +211,56 @@ function joinup_core_module_implements_alter(array &$implementations, $hook) {
       break;
   }
 }
+
+/**
+ * Implements hook_block_access().
+ *
+ * Determines on which pages to show the Webtools eTrans block. Not all pages
+ * are suitable for machine translation.
+ *
+ * This is handled using a custom access check rather than configuring block
+ * visibility conditions in the block UI since we need to check conditions
+ * across several domains. This is not possible in the UI because the conditions
+ * are compared using "AND" logic.
+ *
+ * @see https://www.drupal.org/project/drupal/issues/923934
+ */
+function joinup_core_block_access(BlockInterface $block, string $operation, AccountInterface $account): AccessResultInterface {
+  if ($operation !== 'view' || $block->getPluginId() !== 'oe_webtools_etrans') {
+    return AccessResult::neutral();
+  }
+
+  // Check if we are on a route that is suitable for machine translation.
+  $eligible_route_names = [
+    // The "About" page of a collection or solution.
+    'entity.rdf_entity.about_page',
```

Comment:
How about the Overview page, which is the canonical and landing page?

Reply:
The documentation of eTrans says that it is recommended to use it on pages that are content focused, not on pages which are mainly intended to link to other content. For the overview pages I mainly see these linking to solutions and community content.

It is trivial to add it though. I am sure the functional team will come up with some additional routes where they can be added.

Reply:
This has been escalated to the business analyst team. They decided to show the button only on the About page for now, leaving the option open to add this in the future to the collection/solution homepage if users request it.